cloud/azure/app-gateway/variables.tf

```
@@ -0,0 +1,655 @@
+  type        = list
+  default     = []
+}
+
+variable "unhealthy_host_ratio_notifications_critical" {
+  description = "Notification recipients list for critical alerting rule of unhealthy_host_ratio detector"
+  type        = list
+  default     = []
+}
+
+variable "unhealthy_host_ratio_aggregation_function" {
+  description = "Aggregation function and group by for unhealthy_host_ratio detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".sum(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region', 'backendsettingspool'])"
+}
+
+variable "unhealthy_host_ratio_transformation_function" {
+  description = "Transformation function for unhealthy_host_ratio detector (mean, min, max)"
+  type        = string
+  default     = "max"
+}
+
+variable "unhealthy_host_ratio_transformation_window" {
+  description = "Transformation window for unhealthy_host_ratio detector (i.e. 5m, 20m, 1h, 1d)"
+  type        = string
+  default     = "5m"
+}
+
+variable "unhealthy_host_ratio_threshold_critical" {
+  description = "Critical threshold for unhealthy_host_ratio detector"
+  type        = number
+  default     = 75
+}
+
+variable "unhealthy_host_ratio_threshold_warning" {
+  description = "Warning threshold for unhealthy_host_ratio detector"
+  type        = number
+  default     = 50
+}
+
+# Http_4xx_errors detectors
+
+variable "http_4xx_errors_disabled" {
+  description = "Disable all alerting rules for http_4xx_errors detector"
+  type        = bool
+  default     = null
+}
+
+variable "http_4xx_errors_disabled_critical" {
+  description = "Disable critical alerting rule for http_4xx_errors detector"
+  type        = bool
+  default     = null
+}
+
+variable "http_4xx_errors_disabled_warning" {
+  description = "Disable warning alerting rule for http_4xx_errors detector"
+  type        = bool
+  default     = null
+}
+
+variable "http_4xx_errors_notifications" {
+  description = "Notification recipients list for every alerting rules of http_4xx_errors detector"
+  type        = list
+  default     = []
+}
+
+variable "http_4xx_errors_notifications_warning" {
+  description = "Notification recipients list for warning alerting rule of http_4xx_errors detector"
+  type        = list
+  default     = []
+}
+
+variable "http_4xx_errors_notifications_critical" {
+  description = "Notification recipients list for critical alerting rule of http_4xx_errors detector"
+  type        = list
+  default     = []
+}
+
+variable "http_4xx_errors_aggregation_function" {
+  description = "Aggregation function and group by for http_4xx_errors detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".sum(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
+}
+
+variable "http_4xx_errors_transformation_function" {
+  description = "Transformation function for http_4xx_errors detector (mean, min, max)"
+  type        = string
+  default     = "max"
```

```suggestion
  default     = "min"
```
This transformation was inverted in previous implementation.